arcgis-ios-sdk-samples/Geometry/Geodesic operations/README.md

```diff
@@ -0,0 +1,31 @@
+# Geodesic operations
+
+This sample demonstrates how to use `AGSGeometryEngine` to calculate a geodesic path between two points and measure its distance. Geodesic calculations take into account the curvature of the Earth, while planar calculations are based on a 2D Cartesian plane.
+
+Visualizing flight paths between cities is a common example of a geodesic operation.
+
+![](image1.png)
+
+## How to use the sample
+
+1. Tap on the map to select destination of a path from the origin JFK Airport, NY.
+2. A line graphic will display the geodesic path between origin and destination.
+3. The geodesic distance of the path will display in map view's callout.
+
+## How it works
+
+1. Create an instance of `AGSPolyline` using two points.
+2. Densify the Polyline using the `densifyGeodetic(geometry:maxSegmentLength:lengthUnit:curveType:)` method on `AGSGeometryEngine`. This method will return a geodetically densified geometry.
```

Comment:
Please change 
`densifyGeodetic(geometry:maxSegmentLength:lengthUnit:curveType:)` 
to
`geodeticDensifyGeometry(_:maxSegmentLength:lengthUnit:curveType:)`.

Reply:
Thanks for catching that.